packages/react/src/ReactSharedInternals.js

```diff
@@ -38,12 +38,12 @@ if (__UMD__) {
   // This re-export is only required for UMD bundles;
   // CJS bundles use the shared NPM package.
   Object.assign(ReactSharedInternals, {
-    Schedule: {
+    Scheduler: {
```

Comment:
I think this means a newer React wouldn't work with an older ReactDOM, or vice versa. Since it's scoped to UMDs only I think that's fine and I wouldn't worry about going to extra lengths to make this work for mismatching versions.